query-connector/flyway/sql/V01_07__add_conditions_category.sql

```diff
@@ -0,0 +1,34 @@
+-- 1. Add the new 'category' column to the condition table
+ALTER TABLE conditions
+ADD COLUMN category VARCHAR(255);
```

Comment:
We should probably use `TEXT` instead of `VARCHAR(255)`

[https://www.sprinkledata.com/blogs/postgresql-text-vs-varchar-choosing-the-right-data-type-for-your-database](url)

Reply:
updated!